src/BrewingRecipes.h

```diff
@@ -48,11 +48,14 @@ class cBrewingRecipes
 
 	/** Parses the recipe contained in the line, adds it to m_pState's recipes.
 	Logs a warning to the console on input error. */
-	void AddRecipeFromLine(const AString & a_Line, unsigned int a_LineNum);
+	void AddRecipeFromLine(AString a_Line, unsigned int a_LineNum);
 
 	/** Parses an item string, returns true if successful. */
 	bool ParseItem(const AString & a_String, cItem & a_Item);
 
-	struct sBrewingRecipeState;
-	std::unique_ptr<sBrewingRecipeState> m_pState;
+	using cRecipes = std::vector<std::unique_ptr<cBrewingRecipes::cRecipe>>;
```

Comment:
Move this line to the top of the `private` section. We lay out things in classes: types first, then data members, and functions last.